frontend/src/forms/manage-restaurant-form/ImageSection.tsx

```diff
@@ -0,0 +1,50 @@
+import {
+  FormControl,
+  FormDescription,
+  FormField,
+  FormItem,
+  FormMessage,
+} from '@/components/ui/form';
+import { Input } from '@/components/ui/input';
+import { useFormContext } from 'react-hook-form';
+
+const ImageSection = () => {
+  const { control } = useFormContext();
+
+  return (
+    <div className="space-y-2">
+      <div>
+        <h2 className="text-2xl font-bold">Image</h2>
+        <FormDescription>
+          Add a image that will be displayed on your resataurant listing
+        </FormDescription>
+      </div>
+
+      <div className="flex flex-col gap-8 w-[50%]">
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/omkardongre/Food-Ordering-App/pull/31#issuecomment-2392842420>#ecff14</a><span> - 10/04/2024, 05:21 am</span>
<br>

<div><small>🟠</small> Medium importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The use of a fixed width percentage (w-[50%]) may not be responsive across all device sizes.
</div>
<div id="fix">
<b>Fix:</b> Replace the fixed width percentage with a more responsive approach using Tailwind's responsive classes or a mobile-first design.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -23,1 +23,1 @@
 -      <div className="flex flex-col gap-8 w-[50%]">
 +      <div className="flex flex-col gap-8 w-full md:w-1/2 lg:w-1/3">
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged